src/plugins/event-plugins/__tests__/XhrPlugin.test.ts

```diff
@@ -65,6 +65,7 @@ describe('XhrPlugin tests', () => {
                 statusText: 'OK'
             }
         });
+        expect(plugin.cacheSize).toEqual(0);
```

Comment:
Note: I check this on every test so that there are no edge cases where the cache size net increases

Reply:
Each test should verify one property. Doing this on every test adds unnecessary complexity.